lib/python/picongpu/plugins/data/base_reader.py

```diff
@@ -46,6 +68,52 @@ def get_iterations(self, **kwargs):
 
     def get(self, **kwargs):
         """
+        Parameters
+        ----------
+        Either 'iteration' or 'time' should be present in the kwargs.
+        If both are given, the 'time' argument is converted to
+        an iteration and data for the iteration matching the time
```

Comment:
document how the matching is done.
Internally, you implement the `find_time.get_iteration` default arguments which are:
```
previous : the closest iteration that is <= t (default)
```